src/parser/parser.yy

```diff
@@ -2742,34 +2742,34 @@ drop_hosts_sentence
 
 
 merge_zone_sentence
-    : KW_MERGE KW_ZONE zone_name_list KW_INTO STRING {
+    : KW_MERGE KW_ZONE zone_name_list KW_INTO name_label {
         $$ = new MergeZoneSentence($3, $5);
     }
     ;
 
 drop_zone_sentence
-    : KW_DROP KW_ZONE STRING {
+    : KW_DROP KW_ZONE name_label {
         $$ = new DropZoneSentence($3);
     }
     ;
 
-// split_zone_sentence
-//     : KW_SPLIT KW_ZONE name_label KW_FROM zone_name_list {
-//         $$ = new SplitZoneSentence($3, $5);
-//     }
-//     ;
+split_zone_sentence
+    : KW_DIVIDE KW_ZONE name_label KW_INTO name_label host_list name_label host_list {
```

Comment:
Why only support split into two? Since your merge support merge multiple zone into one, vice versa, you split should support split into multiple zones

Reply:
After talk offline we have reached a consensus on splitting into two zones. How to represent split into multiple part?

Actually, splitting into two items makes sense in most cases